README.md

```diff
@@ -203,10 +203,14 @@ The pattern should something like "base-%03d.html"
 ### hocr-wordfreq
 
 ```
-hocr-wordfreq file.html
+hocr-wordfreq [-h] [-i] [-n MAX] [file.html]
```

Comment:
`[xxx]` is commonly used to indicate optional parts of the command line, and `file.html` is optional. We should replace it by `HOCRFILE` or simply `FILE`, because it is also common to write place holders in upper case. See for example the [man page](http://man7.org/linux/man-pages/man1/cat.1.html) for `cat`.